bin/oscapd-evaluate

```diff
@@ -450,6 +450,18 @@ def main():
         "'docker-container://CONTAINER_ID', 'vm-domain://VM_NAME', "
         "'vm-image:///path/to/image.qcow2', 'chroot:///path/to/chroot'."
     )
+    target_profiles_parser = subparsers.add_parser(
+        "target-profiles",
+        help="Detect security profiles applicable on given target"
```

Comment:
Yes, it is, it lists profiles for SSG. Thanks for suggestion. I will improve the help.